topsort-endpoints.yml

```diff
@@ -152,8 +148,8 @@ components:
         rank:
           type: integer
           description: "Where the product's bid ranked in the auction.
-                        Zero-based, so the product with rank 0 won the auction and had the highest bid.
-                        In an auction response, the winners array is sorted so rank will match the entry's index"
+            Zero-based, so the product with rank 0 won the auction and had the highest bid.
```

Comment:
I don't believe it's zero based.

Reply:
It seems to be https://github.com/Topsort/auction-engine/blob/0f45fd2f4b769c30c87c12898c3d71bffcb34f75/internal/api/auctions_route_test.go#L230

However, this is just a weird design, as I would expect the return array to be sorted by rank. Otherwise it gives the impression that it is not and the client must sort it by themselves.

Reply:
The results are sorted by rank; it's stated explicitly for the convenience of the marketplace -- they can map over the winners and the object knows where it placed instead of having to jam in the index themselves.